ProcessMaker/Managers/ProcessCategoryManager.php

```diff
@@ -7,40 +7,49 @@
 use ProcessMaker\Exception\ValidationException;
```

Comment:
Let's remove this manager completely. Move the validation rules into the model and the business logic into the controller.